pkg/sqlcmd/commands.go

```diff
@@ -282,6 +288,24 @@ func errorCommand(s *Sqlcmd, args []string, line uint) error {
 	return nil
 }
 
+func onerrorCommand(s *Sqlcmd, args []string, line uint) error {
+	if len(args) == 0 || args[0] == "" {
+		return InvalidCommandError("ON ERROR", line)
+	}
+	params := strings.TrimSpace(args[0])
+	if params == "" {
+		s.Connect.ExitOnError = true
+	}
+	if strings.HasPrefix(params, "exit") {
+		s.Connect.ExitOnError = true
+	} else if strings.HasPrefix(params, "ignore") {

```

Comment:
Why not compare the arguments with exact keywords? Is HasPrefix() case sensitive? I think exit or ignore can be specified in any case so we should probably get the arguments to lower case and then compare.